src/main/java/org/folio/circulation/services/LostItemFeeChargingService.java

```diff
@@ -148,11 +157,7 @@ private CompletableFuture<Result<Loan>> closeLoanAsLostAndPaidAndPublishEvent(Lo
 
   private Boolean isOpenLostItemFee(Account account) {
     String type = account.getFeeFineType();
-    if ((type.equals(FeeFine.LOST_ITEM_FEE_TYPE) || type.equals(FeeFine.LOST_ITEM_PROCESSING_FEE_TYPE)) && account.isOpen()) {
-      return true;
-    } else {
-      return false;
-    }
+    return (FeeFine.lostItemFeeTypes().contains(type)) && account.isOpen();
```

Comment:
```suggestion
    return FeeFine.lostItemFeeTypes().contains(type) && account.isOpen();
```

we don't need the brackets

Reply:
Amended, thank you.